src/components/ContentHighlights/ContentHighlightCardItem.jsx

```diff
@@ -18,15 +18,14 @@ const ContentHighlightCardItem = ({
     cardImgSrc: cardImageUrl,
     cardLogoSrc: partners.length === 1 ? partners[0].logoImageUrl : undefined,
     cardLogoAlt: partners.length === 1 ? `${partners[0].name}'s logo` : undefined,
-    cardTitle:
-  <Truncate lines={3} title={title}>{title}</Truncate>,
+    cardTitle: <Truncate lines={3} title={title}>{title}</Truncate>,
     cardSubtitle: partners.map(p => p.name).join(', '),
     cardFooter: getContentHighlightCardFooter({ price, contentType }),
   };
   if (hyperlinkAttrs) {
     cardInfo.cardTitle = (
       <Hyperlink onClick={hyperlinkAttrs.onClick} destination={hyperlinkAttrs.href} target={hyperlinkAttrs.target} data-testid="hyperlink-title">
-        <Truncate lines={3} title={title}>{title}</Truncate>,
+        <Truncate elementType="span" lines={3} title={title}>{title}</Truncate>
```

Comment:
[curious] Why did the `elementType` need to change?

Reply:
Before the change, the external hyperlink icon would always appear on the next line of the component even if it could fit on the same line. By setting the component element type to a span, it allows it the icon to stay on the same line so long as there is sufficient space on the line. 
![image](https://github.com/openedx/frontend-app-admin-portal/assets/82611798/46fa1ef3-891c-4f26-9f4f-ce1f4a91cb04)